### ISSUE TITLE
Content Manager role

### ISSUE THREAD

**Issue by brashrebel** [realbig/TH_GENFINPLAN#2]
Just like on the HVAC site, we need another role created here. You can make another plugin or modify the existing plugin on the same network to be more generic so that it can work for both of these sites.

Once done, assign this new role to user: loren.

**Comment by BigActual** [comment]
Loren has specifically mentioned the ability to edit the states he operates in, listed on the home page. Currently that's in a widget. Also he wants to edit the statement below the slider and the action button as listed in #3. He wants to be able to edit the site tag-line and site title as well. I bring these up to ensure that the role created has the ability to edit these items.

**Comment by d4mation** [comment]
I opted for making a separate plugin in this case in the event that we need to add/remove more Capabilities to the Role.

`content_manager` has all the same Capabilities as `editor` with the removal of `unfiltered_html` and the addition of `edit_theme_options`

This means they have access to edit the Widgets that hold the information about the States Loren operates in. I also made it so that editing `blogname` and `blogdescription` (Site Title and Tagline) from the Customizer requires only `edit_theme_options` rather than `manage_options` like it normally does.

I made all the CTA editing possible in the Customizer within its own section (That only shows if you're viewing the Home Page) as that was much easier (and more maintainable going into the future) than trying to section off a single panel of WooTheme's Theme Options menu.

Changes can be seen in 5a86ad0

Edit: Loren's User has also been reassigned from `editor` to `content_manager`.
